src/Categories.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function Categories({ categories, selectedCatId, onClick }) {
+  return ((
+    <ul>
+      {categories.map((category) => (
+        <li
+          key={category.id}
+        >
```

Comment:
```suggestion
        <li key={category.id}>
```

`props` 가 하나인경우 한줄로 해주는게  더 코드 가독성이 좋을 것 같아요!